### ISSUE TITLE
Save/Update/Fork links not working when the phrase "make the reader" is in the HTML section.

### ISSUE THREAD

**Issue by Tyler-H** [jsfiddle/jsfiddle-issues#1196]
Today I tried to create a JSFiddle and could not save it. I was able to repeat this behavior in Firefox 58 and Chrome 64, both with Ad blocking enabled and disabled. However, sometimes I was able to save a Fiddle. It turns out there is actually one specific fragment of text that, when present in the HTML field, prevents the Fiddle from saving/updating/forking. Trying to press any of those buttons when this text fragment is present simply reloads the frame and changes the URL to https://jsfiddle.net, even if the URL was previously a unique Fiddle string and/or revision number.

The string is:

> make the reader

in that specific order. All other variants of the phrase (other orders, two of the three words, caps lock, inserting an `&nbsp;` in place of one of the spaces) work without issue.

What's going on here?

**Comment by oskarkrawczyk** [comment]
For some reason the anti-spam mechanism is picking this string as rouge.

Will investigate.

**Comment by NOVALISTIC** [comment]
Only reproducible when not authenticated. Works correctly when authenticated.

**Comment by NOVALISTIC** [comment]
@oskarkrawczyk Well... that explains why it doesn't affect authenticated users.

**Comment by oskarkrawczyk** [comment]
Yep, the anti-spam is a lot more restrictive for anonymous users

**Comment by Tyler-H** [comment]
Ahh, I hadn't tried logging in. Was trying to make a brief demo for Stack Overflow so I wasn't super concerned with tying it to my personal library of Fiddles

**Comment by oskarkrawczyk** [comment]
@Tyler-H Sorry about that. We had to increase the anti-spam sensitivity because of the recent ethereum scam frenzy using our service to scam people out of crypto.

That said, it is odd that this particular string is triggering the mechanism.

**Comment by Tyler-H** [comment]
Good old cryptocurrency... ruining life for gamers, now for coders :-P

**Comment by oskarkrawczyk** [comment]
@Tyler-H tell me about it, crypto scammers first used JSFiddle to post fiddles, then DDoSed us when we've increased anti-scam measures.

We just can't have nice things...
